app/containers/Forms/EmissionGasFields.tsx

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
 import {FieldProps} from 'react-jsonschema-form';
 import {Form, Col} from 'react-bootstrap';
+import {JSONSchema6} from 'json-schema';
+import NumberFormat from 'react-number-format';
 
-const EmissionGasFields: React.FunctionComponent<FieldProps> = ({
+interface Props extends FieldProps {
+  annualEmission?: any;
+}
```

Comment:
Doesn't look like you need this. You can just use `FieldProps` below

Reply:
Updated